documentation/docs/developer_guide/obsolete_and_deprecated_features/deprecated_features.md

```diff
@@ -0,0 +1,36 @@
+# Deprecated features for CABLE developers #
+
+Coding with the deprecated developer features described here is discouraged as they are scheduled to become obsolete
+and removed from future versions of the CABLE code. 
+
+Related content which might be of interest:
+
+  - [Obsolete developer features](./obsolete_features.md)
+  - [Deprecated user features](../../user_guide/obsolete_and_deprecated_features/deprecated_features.md)
+  - [Obsolete user features](../../user_guide/obsolete_and_deprecated_features/obsolete_features.md)
+  
+The template below can be used to report deprecated developer features (one feature per block):
+
+## List of deprecated developer features ##
+
+-------------------------------------------------------------------------------
+
+**Deprecated developer feature:** enter the exact syntax if appropriate
+
+**Deprecated from CABLE version:** enter version number and date
+
+**Replacement developer feature?:** describe the current alternative feature
+
+**Comments:** e.g., describe the locations of the deprecated and replacement features and their purposes
+
+-------------------------------------------------------------------------------
+
+**Deprecated developer feature:** enter the exact syntax if appropriate
+
+**Deprecated from CABLE version:** enter version number and date
+
+**Replacement developer feature?:** describe the current alternative feature
+
+**Comments:** e.g., describe the locations of the deprecated and replacement features and their purposes
+
+-------------------------------------------------------------------------------
```

Comment:
I would prefer the following format:

```
| Feature name | <div style="width:150px"> Reason for deprecation </div> | Replacement? | CABLE version | Pull Request |
| ------------ | ------------------------------------------------------- | ------------ | ------------- | ------------ |
| enter the exact syntax of the feature | short reason for deprecation | add the current alternative if any | enter the version number | [#enter the pull request number](copy PR's URL) |
| enter the exact syntax of the feature | short reason for deprecation | add the current alternative if any | enter the version number | [#enter the pull request number](copy PR's URL) |
```

I know you are worried about having a very long table. But otherwise, we will have an even longer page.

Reply:
I have adopted your format